PowerEditor/src/functionList.xml

```diff
@@ -318,13 +318,13 @@
 			<parser
 				id         ="nsis_syntax"
 				displayName="NSIS Syntax"
-				commentExpr="(?s:/\*.*?\*/)|(?m-s:[#;].*?$)"
+				commentExpr="(?s:^[\t\x20]*/\*.*?\*/)|(?m-s:^[\t\x20]*[#;].*?$)"
```

Comment:
The at-start-of-line (`^`) and optional leading blanks (`[\t\x20]*`) don't improve the comment detection but rather make it worse i.e. it prevents detecting trailing comment.